portfolio/pom.xml

```diff
@@ -23,6 +23,31 @@
       <version>4.0.1</version>
       <scope>provided</scope>
     </dependency>
+    <!--
```

Comment:
Do you plan to use this dependency in the future, if not, I suggest we remove it

Reply:
I was trying it out for building wordclouds out of comments, i'm make sure to remove it if i dont use it!